dexbot/strategies/staggered_orders.py

```diff
@@ -309,6 +315,25 @@ def log_maintenance_time(self):
         delta = datetime.now() - self.start
         self.log.debug('Maintenance execution took: {:.2f} seconds'.format(delta.total_seconds()))
 
+    def get_order_creation_fee(self, fee_asset):
+        """ Returns the cost of creating an order in the asset specified
+
+            :param fee_asset: QUOTE, BASE, BTS, or any other
+            :return:
+        """
+        # Get fee
+        fees = self.dex.returnFees()
+        limit_order_create = fees['limit_order_create']
+
+        if fee_asset['id'] == '1.3.0':
+            # Fee asset is BTS, so no further calculations are needed
+            return limit_order_create['fee']
+        else:
+            # Determine how many fee_asset is needed for core-exchange
+            temp_market = Market(base=fee_asset, quote=Asset('1.3.0'))
+            core_exchange_rate = temp_market.ticker()['core_exchange_rate']
+            return limit_order_create['fee'] * core_exchange_rate['base']['amount']
+
```

Comment:
Could this be in base and basestrategy?

Reply:
Yeah I think it's good to move to the base.py later along with migration of Staggered Orders to base.py.

Reply:
I will wait for that change before making this merge.

Reply:
@joelvai ok, changes pushed, please check.